tests/test_crossbar.py

```diff
@@ -6,18 +6,19 @@
 
 import pytest
 import pexpect
-import psutil
```

Comment:
Why not use https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency?

Reply:
Because I didn't know this exists.